docs/guided-tour/error-handling/timeout-error/README.md

```diff
@@ -115,6 +131,21 @@ Once the Installation reaches phase `Failed`, apply the corrected one (
 kubectl apply -f <path to installation-fixed.yaml>
 ```
 
-> Note that this fixed version already contains the annotation `landscaper.gardener.cloud/operation: reconcile`, so that Landscaper will start processing it.
+> Note that this fixed version already contains the annotation `landscaper.gardener.cloud/operation: reconcile`, 
+> so that Landscaper will start processing it.
+
+After some time, the phase of the Installation should be `Succeeded` and the ConfigMap deployed by the Helm chart should 
+exist on the target cluster.
+
+
+## Cleanup
+
+To clean up, delete the Installation from the Landscaper resource cluster:
+
+```shell
+kubectl delete inst -n example hello-world
+```
 
-After some time, the phase of the Installation should be `Succeeded` and the ConfigMap deployed by the Helm chart should exist on the target cluster.
\ No newline at end of file
+Note: if the Installation is not yet in a final phase, the deletion process will not start directly. 
+Rather it will wait until the current deployment process has finished. However, if you do not want
+to wait for this, you can **interrupt** the ongoing deployment as described [above](#interrupting-a-deployment).
```

Comment:
Have you added the timeout example to the top level readme of the guided tour?

Reply:
This is not a new example. It is in the error handling section: "Recovering from Errors" > "6. Handling a Timeout Error".